packages/node_modules/webex/src/webex.js

```diff
@@ -22,8 +24,17 @@ require('@webex/plugin-teams');
 require('@webex/plugin-team-memberships');
 require('@webex/plugin-webhooks');
 
+// Packages that were in the meetings bundle
+// require('@webex/plugin-authorization');
```

Comment:
@arun3528 or @chrisadubois can you explain why you're using these packages compared what we were using before

Reply:
I dont remember importing the authorization plugin can you cross verify it its been used anywhere . ?

Reply:
It may be a remnant from the meetings-specific sample build.